### ISSUE TITLE
Relayed API does not coop well with versions

### ISSUE THREAD

**Issue by kvanhijf** [openvstorage/framework#1120]
Initiating a relayed API call from 1 environment to another where the source had API version 6 and the destination had API version 5, resulted in API errors
```
{error_description: "API version requirements: 2 <= <version> <= 5. Got 6", error: "invalid_version"}
error
:
"invalid_version"
error_description
:
"API version requirements: 2 <= <version> <= 5. Got 6"
```

This means that newer installs can never communicate with older installs over the relayed API

**Comment by khenderick** [comment]
The relay should pass raw version instead of translating from the relay's point of view.

tl;du: shouldn't be hard to fix, an hour or two should be sufficient.

**Comment by khenderick** [comment]
Fixed by #1125, packaged in openvstorage-2.7.4-rev.4263.5c2ada6

**Comment by JeffreyDevloo** [comment]
### Information

The issue is that the relaying api version was older than the one we started on. The original client we post to builds up his own client to relay the requested information back through him. This client was however built up using the originals client api version instead of the relaying api's version, resulting in a version mismatch.
Testing using Postman to send api calls.

### Steps
- Updated environment
- Initiated postman
- Logged in using Postman
- Relayed to old environment (version 5)

```
https://{{node}}/api/relay/storagerouters/?ip=10.100.193.154&port=443&client_id=afffa776-d6ea-4f70-a322-d3955992b5f2&client_secret=%2Bwyq%3Et%3B_%7D.%40K~6f-ul9nkC8*_%2CXdz1%3DHT%5B0RJ6pbxvkY9tG6%3C%7DcWn~!%7C%3B87a_kHU
```
- Change headers to other versions
  ### Output

### Test results

#### Expected outputs

When using version 5+, I should receive an identical error as the issue report by my own api client. (Not relayed)

When using lower versions my call should be relayed and I should get an answer

#### Results

##### Version = '*'

![selection_043](https://cloud.githubusercontent.com/assets/17570109/20009825/338b22c0-a2a6-11e6-8e17-ed7e297a0b7c.png)

##### Version = '6'

![selection_044](https://cloud.githubusercontent.com/assets/17570109/20009827/3524bca4-a2a6-11e6-9e52-281d043ee58a.png)
Important to note here is that the Header only say 6 headers.
The key difference is that the API will not relay the call when the version mismatches but it will respond itself.
**Side by side comparison**:
With relay header (normal relay call)
![selection_049](https://cloud.githubusercontent.com/assets/17570109/20010189/7b7a4c04-a2a7-11e6-8fda-ecd652ab650e.png)

Lower is without relay header(version mismatch)
![selection_050](https://cloud.githubusercontent.com/assets/17570109/20010191/7d07e342-a2a7-11e6-9b04-f57bc48ede02.png)

##### Version = '5'

![selection_048](https://cloud.githubusercontent.com/assets/17570109/20010071/1b8d0e3a-a2a7-11e6-9b6f-df0b2458c896.png)

##### Version = '4'

![selection_045](https://cloud.githubusercontent.com/assets/17570109/20009885/72366be2-a2a6-11e6-965b-bf669613ab24.png)

##### Version = '3'

![selection_046](https://cloud.githubusercontent.com/assets/17570109/20009887/73e7b9e6-a2a6-11e6-876f-3cf6370e50f7.png)

### Test result

Since the issue has risen due to the version being '*', it is safe to say that the issue has been resolved due to the latest patch.
Test passed.
